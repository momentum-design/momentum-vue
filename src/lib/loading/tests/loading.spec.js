import { shallowMount } from '@vue/test-utils';
import Loading from '../index.vue';

describe('Loading', () => {
  it('should match snapshot', () => {
    const wrapper = shallowMount(Loading);
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should render 3 span children', () => {
    const wrapper = shallowMount(Loading);
    const spans = wrapper.findAll('span');
    expect(spans.length).toBe(3);
    for (let i=0; i<3; i++) {
      expect(spans.at(i).classes('md-loading__icon')).toBe(true);
    }
  });

});

