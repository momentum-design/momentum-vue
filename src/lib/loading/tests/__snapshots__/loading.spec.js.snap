// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match snapshot 1`] = `<div class="md-loading"><span class="md-loading__icon"></span> <span class="md-loading__icon"></span> <span class="md-loading__icon"></span></div>`;
